internal/proxy/attribute/attribute.go

```diff
@@ -1,12 +1,21 @@
-package middleware
+package attribute
+
+import (
+	"strings"
+
+	"github.com/valyala/fasttemplate"
+)
 
 const (
-	AttributeTypeQuery       AttributeType = "query"
-	AttributeTypeHeader      AttributeType = "header"
 	AttributeTypeJSONPayload AttributeType = "json_payload"
 	AttributeTypeGRPCPayload AttributeType = "grpc_payload"
-	AttributeTypePathParam   AttributeType = "path_param"
+	AttributeTypeQuery       AttributeType = "query"
```

Comment:
We can refactor into something like this
```suggestion
    TypeQuery      Type = "query"
```